### DISCUSSION TITLE
( Blog ) Representing a floating lidar deployment using the WRA data model

### DISCUSSION THREAD

**Discussion by abohara** [IEA-Task-43/digital_wra_data_standard#212] [Ideas]
A central location to discuss topics & developments to help publish a blog that describes how the WRA model could be used to digitally represent a floating lidar deployment campaign. Links to Google drive and other resources will be added here as well as they become available.

-----
Public Lidar data sets in consideration:

- [NYSERDA](https://oswbuoysny.resourcepanorama.dnv.com/download/f67d14ad-07ab-4652-16d2-08d71f257da1): NY Bight. Eolos Platform - ZX lidar
- [A2E](https://a2e.energy.gov/data#ProjectFilter=%5B%22buoy%22%5D): Collection of public data sets. ( Awaiting some specific project recommendation from Cristina @ Vaisala)

**Comment by abohara** [comment]
### 10 Feb 2023 ( Kickoff discussion)
------------------------
Present:  Adria ( Eolos ), Daniel ( Eolos ), Cristina ( Vaisala ) , Amit

**Discussion:**
- Introductions
- Amit provided a brief summary of prior blog ( basic & introductory ) and a sample JSON that represents a MET mast available in the [Repo](https://github.com/IEA-Task-43/digital_wra_data_standard/blob/master/demo_data/floating_lidar_demo_iea43_wra_data_model.json) 
- Discussed viability of the NYSERDA dataset ( link above ) as a possible use case. Adria indicated the EO6 deployment as a good candidate since it had some history of required maintenance / device replacement that could provide a helpful use case to cover in the blog.
- Cristina provided a link to a A2E portal that has other publicly available datasets corresponding to wind cube deployment.
- Amit discussed a request from Andy Clifton to create the data model digital from the raw files. Adria indicated that the raw files may not contain sufficient metadata


**Next steps:**

- [ ] - Amit to start this page as a way to document and keep the conversation going offline.
- [ ] - Amit to start a Google doc to document a skeleton version of the blog that we can collaboratively edit
- [ ] - Adria to look into some raw sample files and any incident reports that may be a good use case to report digitally
- [ ] - Cristina to indicate some specific project(s) that have publicly available WindCube data.

**Comment by cbenzo28** [comment]
Here's two links where you can find WindCube Offshore data from Humboldt and PNNL, it's averaged and filtered data. Let me know if you're looking for raw data, the data below is the standard analyzed output. 

Humboldt: https://a2e.energy.gov/data#AreaFilter=%2240.9733694493073%2C-124.59787845611574%20to%2040.968509014662644%2C-124.58333015441896%22&ProjectFilter=%5B%22buoy%22%5D

PNNL: https://a2e.energy.gov/data#

**Reply by abohara** [reply]
Thank you @cbenzo28

**Comment by AndyBrightWind** [comment]
Hi all, as promised, see below for links to some other publicly available floating LiDAR datasets from the Netherlands:
1. Borselle: https://offshorewind.rvo.nl/files/view/410ac72a-fb0c-4b73-9ebb-ea0afd39cffb/1523020329bwfz_20180315%20borssele%20raw%20lidar%20data-f.zip
2. Ijmuiden (still underway): https://offshorewind.rvo.nl/cms/view/4c4cb8e3-a1c7-404f-917f-a2f6e0d8c803/wind-en-water-ijmuiden-ver
3. Nerdwiek (still underway): https://offshorewind.rvo.nl/cms/view/500ae8a0-e3bc-467f-81fe-9196fe42413a/wind-en-water-nederwiek

**Reply by abohara** [reply]
Thank you @AndyBrightWind

**Comment by abohara** [comment]
### 16 Feb 2023 ( Kickoff discussion)
===================

**Floating Lidar use-case blog ideas discussion:**

Discussed why a blog would be helpful and our prior work in this space.

**Discussed ideas for use cases** that would be well-suited to highlight the capabilities of the data model. Some ideas were:
- Device swaps - _which has occurred in the NYSERDA data and hence may be the main example to highlight in this blog_
- Any changes to device platform height 
- Bearing offset change e.g. corrections made if the optical axis was not point north ( _Though more of an onshore use case_ )
- Internal parameter changes - _though this may not always be published_
- Coordinate drift - _in the offshore use case this is a measurement rather than metadata_
- Source of heading / true north - _this may change automatically, in which case it may be a measurement than a metadata_

**Conversations around public datasets:**
- NYSERDA has considered so far ( ZX lidars and has device changes - so it is a reasonable data set )
- Also, discussed some Wind Cube deployments as their logger files have additional metadata in the file header that can ease the generation of the data model JSON file
- More public data sets will be suggested by Andrew 
- Cristina will help with identifying more information on some of the Wind Cube data publicly

**Discussed having simple but impactful visualization** to end the blog. Some ideas below:
- Discussed a small interactive widget that renders a lidar from the JSON (_Amit_)
- Timeline visualization of overall device swap schedule overlaid on measurements . Might provide context for users who like to see if the measurements are impacted by swaps ( _Cristina_ )
- A way to represent device swaps ( _Stephen_ )
- An example that represents how much time can be saved by digesting a visualization generated from the digital data file as opposed to reading text / pdf ( _Salma/Lear @ Arkocean_)
